v2/config/manager/manager.yaml

```diff
@@ -38,7 +38,7 @@ spec:
       serviceAccountName: default
       containers:
       - args:
-        - --metrics-addr=:8443
+        - --metrics-addr=0.0.0.0:8443
```

Comment:
minor: Can you explain why we need this change?

Reply:
Ah, it's because that's the default in the Helm chart, right?

Reply:
Yep